buildenv/jenkins/openjdk_tests

```diff
@@ -342,6 +342,12 @@ timestamps{
                     }
                 }
 
+            if (PLATFORM == "ppc64_aix" && env.JENKINS_URL.contains("hyc-runtimes")) {
```

Comment:
This isn't quite right as it still applies the LABEL addition even if the LABEL parameter in the grinder is set. Do you know how to check for that?

Reply:
I think I've got it.